code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -319,6 +323,7 @@
 	allowed = list()
 	actions_types = list(/datum/action/item_action/toggle_human_head)
 	hoodtype = /obj/item/clothing/head/hooded/human_head
+	species_exception = list(/datum/species/golem) //Finally, flesh
```

Comment:
niiiiice

by the way, have you tested these to make sure that the overlays and such from these various pieces of clothing play nice with golem sprites?

Reply:
I don't see any reason why they wouldn't. If the labcoats work, which have no particularly special overlay business going on, then most any other sprite should.